### PR TITLE
Namespace and reorganise components location

### PR DESCRIPTION
**Pull request by Davidslv**
This PR is to properly namespace both StepNavHelper and RelatedNavigationHelper to be under GovukPublishingComponents namespace.

It also corrects the spec locations.

cc\ @emmabeynon @sihugh

### CODE REVIEW

**Comment by sihugh** [comment]
I'm happy if @emmabeynon 's happy

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+** Unreleased **
+
+Properly namespaces StepNavHelper and RelatedNavigationHelper to be under GovukPublishingComponents namespace.
+This is a breaking change to all the apps using this two components.
```
Comment (tijmenb) [inline CHANGELOG.md]: Why is this breaking? Aren't these classes only used from inside the component?

**Comment by emmabeynon** [comment]
This seems fine to me.  I think @tijmenb is correct that it shouldn't be breaking.

**Comment by Davidslv** [comment]
Thank you, I got confused with the classes that were exposed. I've removed the line in the CHANGELOG.md

File: lib/govuk_publishing_components/components/step_by_step_nav_helper.rb [hunk 2]
```diff
@@ -1,107 +1,111 @@
-class StepNavHelper
-  include ActionView::Helpers
-  include ActionView::Context
+module GovukPublishingComponents
+  module Components
+    class StepNavHelper
```
Comment (Davidslv) [inline lib/govuk_publishing_components/components/step_by_step_nav_helper.rb]: this class name is wrong according to the file name (I've just noticed this)

**Comment by tijmenb** [comment]
This is superseded by https://github.com/alphagov/govuk_publishing_components/pull/206. Thanks @Davidslv!

**Pull request closed without merging**